lms/static/sass/base/_variables.scss

```diff
@@ -134,7 +134,8 @@ $green-u2: desaturate($green,30%);
 $green-u3: desaturate($green,45%);
 
 // TODO: both blue and yellow variables differ from CMS rgb value, need to confirm change to CMS variable is ok in current platform uses before switching.
-$blue: rgb(29,157,217);
+$blue: rgb(0, 125, 184); // #007db8 // AA compliant
+$blue-on-gray: rgb(19, 115, 161); // #1373a1 // AA compliant
```

Comment:
TLDR: remove $blue-on-gray and make $blue: rgb(19, 115, 161);

I'm not sure about this variable - I would prefer we made the base blue this color and not need to wire through a specific color match. Once we have patterns, we can bake the contrast through in a more logical way.